kubernetes/docs/AppsV1beta1Deployment.md

```diff
@@ -3,11 +3,11 @@
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**ApiVersion** | **string** | APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources | [optional] [default to null]
-**Kind** | **string** | Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds | [optional] [default to null]
-**Metadata** | [***V1ObjectMeta**](v1.ObjectMeta.md) | Standard object metadata. | [optional] [default to null]
-**Spec** | [***AppsV1beta1DeploymentSpec**](apps.v1beta1.DeploymentSpec.md) | Specification of the desired behavior of the Deployment. | [optional] [default to null]
-**Status** | [***AppsV1beta1DeploymentStatus**](apps.v1beta1.DeploymentStatus.md) | Most recently observed status of the Deployment. | [optional] [default to null]
```

Comment:
so the description is removed, any idea what's the reason for that?

Reply:
as above.